fetch.bs

```diff
@@ -6892,6 +6900,7 @@ dictionary RequestInit {
   boolean keepalive;
   AbortSignal? signal;
   RequestDuplex duplex;
+  FetchPriority priority;
```

Comment:
I'd be somewhat inclined to name this `RequestPriority` to match the others. It doesn't matter much as it's not exposed, but still.

Reply:
Done